build.gradle

```diff
@@ -15,17 +16,17 @@ app {
 }
 
 dependencies {
-  compile "com.enonic.xp:core-api:${xpVersion}"
-  compile "com.enonic.xp:portal-api:${xpVersion}"
+  implementation "com.enonic.xp:core-api:${xpVersion}"
+  implementation "com.enonic.xp:portal-api:${xpVersion}"
   include "com.enonic.xp:lib-event:${xpVersion}"
   include "com.enonic.xp:lib-auth:${xpVersion}"
   include "com.enonic.xp:lib-context:${xpVersion}"
   include "com.enonic.xp:lib-io:${xpVersion}"
   include "com.enonic.xp:lib-portal:${xpVersion}"
   include "com.enonic.xp:lib-common:${xpVersion}"
-  include 'com.enonic.lib:lib-http-client:2.4.1'
-  include 'com.enonic.lib:text-encoding:1.1.0'
-  include 'com.enonic.lib:lib-util:2.0.0'
+  include 'com.enonic.lib:lib-http-client:3.2.1'
+  include 'com.enonic.lib:lib-text-encoding:2.1.0'
+  include 'com.enonic.lib:lib-util:3.0.0'
```

Comment:
I have held back doing these updates myself because of fear of breakages (and not enough time to experiment).

If you can confirm that the application continues working, I'll merge this PR in.

Reply:
I see. I have only tested it the way I use it and I hadn't noticed any bugs yet, but found one now. I'll see when I have time to do a more thorough testing, and hopefully fix the bug!

Reply:
The bug I found here as well, is fixed in https://github.com/ItemConsulting/app-azure-ad-idprovider/pull/8 (I talked with Anton about this as well before creating that PR).

Reply:
And after that bugfix, everything works as expected for me. I have tested:

- Different config for "User Mappings" ("Name", "Display Name" and "Email")
- With "Create and update groups from graph api" checked
- Different config for "The page return size from graph api"
- With one "Group Filter" ("Property", "Regexp")

And I am able to log in and get groups as expected.

But, I have not tested:

- Without checked "Create and update groups from graph api"
- More than one "Group Filter" nor the "AND" checkmark in this config
- With "Force the redirect uri to use https" checked
- Proxy

Reply:
Sound great! :+1: